x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/components/analytics_section.tsx

```diff
@@ -7,18 +7,38 @@
 
 import React from 'react';
 
-import { EuiPageContentBody, EuiTitle, EuiText, EuiSpacer } from '@elastic/eui';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiPageContentBody,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  IconType,
+} from '@elastic/eui';
 
 interface Props {
   title: string;
   subtitle: string;
+  iconType?: IconType;
 }
-export const AnalyticsSection: React.FC<Props> = ({ title, subtitle, children }) => (
+export const AnalyticsSection: React.FC<Props> = ({ title, subtitle, iconType, children }) => (
   <section>
     <header>
-      <EuiTitle size="m">
-        <h2>{title}</h2>
-      </EuiTitle>
+      <EuiFlexGroup gutterSize="xs" alignItems="center" justifyContent="flexStart">
+        {iconType ? (
+          <EuiFlexItem grow={false}>
+            <EuiIcon type={iconType} size="l" />
+          </EuiFlexItem>
+        ) : null}
```

Comment:
This can just be a `&&` conditional instead of a ternary

```suggestion
        {iconType && (
          <EuiFlexItem grow={false}>
            <EuiIcon type={iconType} size="l" />
          </EuiFlexItem>
        )}
```

Reply:
https://github.com/elastic/kibana/pull/92939/commits/71a5d661148b1bb17ef57b1a48355606c6587c3a